### ISSUE TITLE
so version has not been upgraded to 2.9.5

### ISSUE THREAD

**Issue by Lesseps** [libfuse/libfuse#10]
Previous releases of fuse (libfuse) had package and so versions identical:

```
fuse-2.9.x -> libfuse.so.2.9.x
```

In present version 2.9.5, should be **libfuse.so.2.9.5**, not libfuse.so.2.9.4 . This has not been done.

It is defined in _lib/Makefile.am_, and should be

```
libfuse_la_LDFLAGS = -pthread @libfuse_libs@ -version-number 2:9:5 \
```

not

```
libfuse_la_LDFLAGS = -pthread @libfuse_libs@ -version-number 2:9:4 \
```

For users, fix just changing _lib/Makefile.in_ with

```
sed -i  's/2:9:4/2:9:5/' lib/Makefile.in
```
For the package, tested the same with _lib/Makefile.am_:

```
sed -i  's/2:9:4/2:9:5/' lib/Makefile.am
```
but needed to _autoreconf_.

Please, tell me if you need any other info.

Thanks.

**Comment by Nikratio** [comment]
Thanks for the extensive report! I'll make sure to do this correctly in the future!

**Comment by Nikratio** [comment]
(I don't think I need to update this release, because there were absolutely no changes to the API)

**Comment by Nikratio** [comment]
Opening this again to prevent further duplicates.

**Comment by Nikratio** [comment]
Fixed in 2.9.6.
